### PR TITLE
feat(amplify-frontend-flutter)

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Adds developer preview functionality for amplify-flutter projects.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/scripts/post-install.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ console.log(chalk.green('Android Getting Started - https://docs.amplify.aws/star
 console.log(EOL);
 console.log(chalk.green('iOS Getting Started - https://docs.amplify.aws/start/q/integration/ios'));
 console.log(EOL);
+console.log(chalk.green('Flutter Getting Started - https://docs.amplify.aws/start/q/integration/flutter'));
```
Comment (renebrandel) [inline packages/amplify-cli/scripts/post-install.js:28]: Flutter (Developer Preview)

Comment (haverchuck) [reply packages/amplify-cli/scripts/post-install.js:28]: Done!

**Review by attilah — CHANGES_REQUESTED** [review]
I reviewed the other Flutter PR, those comments should be addressed here then ;-)

**Comment by haverchuck** [comment]
@attilah I'm assuming you mean this one:  https://github.com/aws-amplify/amplify-cli/pull/5775

**Comment by attilah** [comment]
@haverchuck yes.

**Comment by haverchuck** [comment]
@attilah I answerd your inline comments in @AaronZyLee 's  initial PR.  I made several of your requested revisions.

You also had two general comments:

> * Put all the Flutter related E2E tests into 1 test file within the E2E project to make separation easier later and don't spread across categories.
> * Reorg the plugin source code into `src` folder with `lib` output and add TypeScript passthrough compilation, like we have in other some packages like `amplify-provider-awscloudformation` as an example.

Regarding the first bullet, I decided not to do this because none of the other test files appear organzied by platform.  I think it would be better to reorganzie all of them if needed.

Regarding bullet 2 - as we discussed, this would potentially be a breaking change due to the need to upgrade the graphql-config library to version 3.  This upgrade would break our amplify-codegen package, since the new graphql-config libary uses "include/exclude" instead of "includes/excludes".

**Review by attilah — APPROVED** [review]
LGTM, thanks for the changes, as we discussed, I understand the TS concerns, we can address it later for all the packages in 1 go.

**Pull request merged**